action-publish-code/publish-code.sh

```diff
@@ -55,13 +61,18 @@ done
 exit 0
 
 
-for repo in "$@"; do
+for arg in "$@"; do
+	dir="${arg%:*}"
+	repo="${dir##*/}"
+	if [[ "$arg" != *":"* ]]; then
+		repo="${arg##*:}"
```

Comment:
if this was golang, I'd suggest avoiding re-doing this work that was also done above.

But since its bash, copy-pasta may be easier :)

Reply:
Hehe, yeah. FWIW, eventually I might rewrite in `go`. But it's working now so ....